scss/components/_stickybar.scss

```diff
@@ -8,16 +8,24 @@
 // Styleguide components.sticky-bar
 
 .sticky-bar {
-  @include transition(top .4s ease-out);
-  top: -200px; // Setting this property so we can animate it when .is-stuck is applied
+  // @include transition(top .4s ease-out);
   z-index: $z-sticky;
   width: 100%;
 
   // To stick to the top of the page
   &.is-stuck {
-    @include transition(top .4s ease-out);
     position: fixed;
     top: 0;
     left: 0;
   }
+
+  &.with-transition {
+    @include transition(top .4s ease-out);
+    top: -200px; // Setting this property so we can animate it when .is-stuck is applied
+
+    &.is-stuck {
+      @include transition(top .4s ease-out);
```

Comment:
I don't think you need this, since the `with-transition` class already has the transition property.